contracts/src/bridge/Outbox.sol

```diff
@@ -182,8 +182,12 @@ contract Outbox is DelegateCallAware, IOutbox {
         bytes32 calcRoot = calculateMerkleRoot(proof, index, item);
         if (roots[calcRoot] == bytes32(0)) revert UnknownRoot(calcRoot);
 
-        if (spent[index]) revert AlreadySpent(index);
-        spent[index] = true;
+        uint256 spentIndex = index / 255; // Note: Reserves the MSB.
+        uint256 bitOffset = index - spentIndex * 255;
```

Comment:
One minor change: can you compute `bitOffset` with `index % 255`? Other than that, LGTM

Reply:
Ahh you're right, that's simpler and should save 6 gas. Just updated that